src/flatc_main.cpp

```diff
@@ -107,7 +107,7 @@ int main(int argc, const char *argv[]) {
     { flatbuffers::GenerateJsonSchema, nullptr, "--jsonschema", "JsonSchema",
       true, nullptr, flatbuffers::IDLOptions::kJsonSchema,
       "Generate Json schema", nullptr },
-    { flatbuffers::GenerateSwift, nullptr, "--swift", "swift", true,
+    { flatbuffers::GenerateSwift, "-S", "--swift", "swift", true,
```

Comment:
a capital option is a bit different from the existing options, and I'd prefer to keep it to only long options from now on.. I'd remove it for other languages if it weren't for backwards compatibility.